pallets/gear/src/weights.rs

```diff
@@ -19,7 +19,7 @@
 //! Autogenerated weights for pallet_gear
```

Comment:
Comments addressed in runtime/vara/src/weights/pallet_gear.rs is also applicable here.